modules/hal_nordic/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ if(CONFIG_NRF_REGTOOL_GENERATE_UICR)
   list(APPEND nrf_regtool_components GENERATE:UICR)
 endif()
 if(DEFINED nrf_regtool_components)
-  find_package(nrf-regtool 5.6.0 REQUIRED
+  find_package(nrf-regtool 6.0.0 EXACT REQUIRED
```

Comment:
Is it really needed to require exactly this version?

Reply:
No you are right it is not, I dropped the commit that bumped the version, it can be done independently later